src/Action/Lti/GetUserAssignmentLtiLinkAction.php

```diff
@@ -88,7 +88,7 @@ public function __invoke(UserInterface $user, int $assignmentId): Response
             return $this->responder->createJsonResponse($ltiRequest);
         } catch (AssignmentNotFoundException $exception) {
             throw new NotFoundHttpException($exception->getMessage());
-        } catch (AssignmentUnavailableException | AssignmentNotProcessableException $exception) {
+        } catch (AssignmentUnavailableException $exception) {
```

Comment:
Why was it removed?

Reply:
To reduce coupling I merged the two exceptions, there are not much difference between them.